gandalf/src/main/java/io/github/btkelly/gandalf/utils/LoggerUtil.java

```diff
@@ -23,26 +23,35 @@
 public final class LoggerUtil {
 
     private static final String LOGGER_TAG = "Gandalf";
+    private static LogLevel level = LogLevel.NONE;
 
     public enum LogLevel {
```

Comment:
If we do an int def will the parameter still auto complete to the types when using the Gandalf installer? If setting the log level just becomes an integer I would like to keep it as an enum.